tests/func/test_update.py

```diff
@@ -51,6 +110,48 @@ def test_update_import(tmp_dir, dvc, erepo_dir, cached):
     }
 
 
+def test_update_before_and_after_dvc_init(tmp_dir, dvc, erepo_dir):
+    with erepo_dir.chdir():
+        erepo_dir.scm.repo.index.remove([".dvc"], r=True)
+        shutil.rmtree(".dvc")
+        erepo_dir.scm_gen("file", "first version")
+        erepo_dir.scm.add(["file"])
+        erepo_dir.scm.commit("first version")
+        old_rev = erepo_dir.scm.get_rev()
+
+    stage = dvc.imp(fspath(erepo_dir), "file", "file")
+
+    with erepo_dir.chdir():
+        Repo.init()
```

Comment:
Hm, not sure why this is needed. erepo is already a git repo. Am I missing something?

Reply:
`Repo` here is DVC. I'm enabling DVC on the git repo. In the first part of the test I delete `.dvc` to test the case where an external repo gets `dvc init`'ed after an import.